vizro-ai/src/vizro_ai/chains/_llm_models.py

```diff
@@ -56,33 +35,25 @@ class ModelConstructor:
     Provides methods to fetch LLM details and instantiate appropriate wrappers.
     """
 
-    models: Dict[str, LLM]
-
-    def __init__(self):
-        """Initializes the model manager with a set of predefined LLMs."""
-        self.models = {model["name"]: LLM(**model) for model in PREDEFINED_MODELS}
-
-    def get_llm_model(self, model_name: str, temperature: float = 0) -> LLM_MODELS:
+    def get_llm_model(self, model=None) -> LLM_MODELS:
```

Comment:
Isn't this a breaking change? People who have been using these args before - it wouldn't work for them anymore right? I don't know the context of it, but why do we introduce this breaking change and remove these args?